report_thesis/src/sections/background/data_overview.tex

```diff
@@ -21,4 +21,7 @@ \subsection{Data Overview}\label{sec:data-overview}
 \end{table*}
 
 While the \gls{ccs} data is in a more suitable form for quantitative analysis, it still requires further preprocessing. 
-Table~\ref{tab:ccs_data_example} shows an example of the \gls{ccs} data for a single location of a sample. This corresponds to shots ($s$) and wavelength ($\lambda$) of the Intensity Tensor \ref{matrix:intensity} for this location.
\ No newline at end of file
+Table~\ref{tab:ccs_data_example} shows an example of the \gls{ccs} data for a single location of a sample. This corresponds to shots ($s$) and wavelength ($\lambda$) of the Intensity Tensor \ref{matrix:intensity} for this location.
+The \gls{ccs} data will at this stage have negative values and noise at the edges of the spectrometers.
+How these are handled is described in Section~\ref{sec:data-preparation}.
+Once prepared, the \gls{ccs} data can now be used for preprocessing and model training.
```

Comment:
> Describes preprocessing
> "Now it's ready for preprocessing"

I think this could be better integrated with the sentence above.
I gave a suggestion for it :)